vagga.yaml

```diff
@@ -1,26 +1,87 @@
+minimum-vagga: v0.7.0-1-gc613573
```

Comment:
I don't think vagga is able to compare commit hash part of the version, please cut it off.